### PR TITLE
Made changes to Team Page Headers, sorted team by last name

### LINKED ISSUE #45: Update team to be alphabetical on last name
We should update the team page to be listed alphabetical on last name

The headings could also be tweaked:

Core Team = Core Contributors
Members = GitHub Members

Then let's add a note to the README.md to say how to add a new person based upon the instructions from @johnmccabe in #44 

Alex

### PR DESCRIPTION
**Pull request by saldistefano**
Changed Headers in Team Page, ordered team by last name and updated README.md with instructions on how to add team members.

Fixes #45

Signed-off-by: Sal DiStefano <sal@saldistefano.com>

### CODE REVIEW

**Comment by burtonr** [comment]
@saldistefano Why did you close this? It hasn't been merged yet. I only left my review, @alexellis will need to do the merge

**Comment by saldistefano** [comment]
Thanks @burtonr, I've reopened.

Sal

File: README.md [hunk 1]
```diff
@@ -87,9 +87,11 @@ For example, if you add your own post in `_posts/2018-07-31-my-post.md` you will
     openfaas-jekyll_1  |       Remote Theme: Using theme cloudcannon/frisco-jekyll-template
     openfaas-jekyll_1  |                     ...done in 2.0199265 seconds.
 
-### Adding an Author
+### Adding an Author\Team Member
```
Comment (alexellis) [inline README.md]: Can this be a forward slash please?

File: README.md [hunk 2]
```diff
@@ -87,9 +87,11 @@ For example, if you add your own post in `_posts/2018-07-31-my-post.md` you will
...
 
-In order to have a blog post properly attributed to an author you must create an `<author_name>.md` file in the `_staff_members` directory.
+In order to have a blog post properly attributed to an author, they must be added to team by creating an `<author_name>.md` file in the `_staff_members` directory.
+
+Display of team member on Team Page is based on position in the data/members.yml file.
```
Comment (alexellis) [inline README.md]: Could this be a link?

**Comment by alexellis** [comment]
Couple of comments for a follow-up PR/commit.

Thanks for the fast turnaround. 

Alex

**Pull request merged**